### PR TITLE
Add "Design Thinking" section and detail current user personas

### LINKED ISSUE #126: User Personas & Design Thinking
### Epic

_No response_

### Description

Have a section in our handbook that details the results of our recent in-house persona workshops. Include profiles for each persona, who they are, interests, etc.

Then discuss a bit on the philosophy of personas/design thinking and why we use them. 

### Acceptance Criteria

- [ ] json driven personas for easy editing
- [ ] visually appealing profiles, not just text, to make them more consumable

### PR DESCRIPTION
**Pull request by joepavitt**
Closes #126 

~Ensure https://github.com/flowforge/website/pull/233 is merged first before merging this to prevent odd image styling~

https://user-images.githubusercontent.com/99246719/199553611-5a0c733a-7ce4-47c1-aa08-dbff38bf23d4.mov

### CODE REVIEW

File: design/design-thinking.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Personas are fictional characters that represent the different users we expect to interact with FlowForge as a brand and/or platform. You can find samples of the user personas we have at FlowForge in our [Design Thinking Output](./design-thinking).
+
+At FlowForge, we have many personas that will evolve over time. To give you an insight into what we are working with currently, here is a sample of our 5 MVPs (Most Valuable Personas) to-date:
+
+<div class="grid grid-cols-2 gap-4">
+    <div class="rounded-md bg-gray-900 text-white p-4 border-2 border-black">
+        <div class="py-2 text-center border-b-2 border-red-400">
+            <img class="w-32 rounded-md border-2 border-white m-auto" src="../images/personas/harry.jpg">
+            <label class="block mt-2 font-bold">Harry, the Hobbyist</label>
```
Comment (ZJvandeWeg) [inline design/design-thinking.md:13]: I'd rather not focus on Hobbyists at this time of the company. I'd like to remove this persona completely, can we?

File: design/design-thinking.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        </div>
+        <div class="py-2">
+            <div class="flex flex-col">
```
[the remark this answers is not in the document]
Comment (joepavitt) [reply design/design-thinking.md:16]: The point of a persona is that you define detail, age is an important factor for a persona. It's not to say we won't have any Consultants the age we specify, but it does give us a ballpark as to the general age we would come across. This is important, as age is an influential factor when defining UX and targeting different users with content in socials for instance.

File: design/design-thinking.md (hunk 1 above)
Comment (joepavitt) [reply design/design-thinking.md:13]: Hobbyists are already the largest portion of the user base for FlowForge, to not include them as an MVP would be an error in my opinion. Whilst they may not bring in the largest amount of money, we still need to respect their opinion and considerations when in some capacity.

Comment (joepavitt) [reply design/design-thinking.md:13]: In the workshop we ran, we concluded that an important persona would be a crossover between Harry and Neil. Someone that would use FlowForge to run a flow or two for some of their devices, but also contributed to their own Node-RED flows - I am happy to create this persona as a standalone, and merge Neil/Harry together.

Comment (joepavitt) [reply design/design-thinking.md:13]: I am happy to re-order them, as there is some unconscious assessment of priority when reading them, which whilst isn't intentional, is unavoidable for a reader of the content.

File: design/design-thinking.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ZJvandeWeg) [reply design/design-thinking.md:16]: I fear some form of age discrimination, the numbers here and during the workshop felt artificial, especially as there's an AI generated face next to them. Did we first generate a face, or the age, and does that influence what we develop for the persona?

And while I understand that age influences UX, I don't see the difference between a developer from about 30, or one aged about 60?

[the remark this answers is not in the document]
Comment (joepavitt) [reply design/design-thinking.md:16]: Having worked with many developers in their 60s, and many in their 30s - the difference is quite stark. Even between devs in their 30s and devs in their early 20s I have found a big difference. They've grown up in drastically different technical ecosystems, with very different expectations as to the effort required to achieve technological goals - this has huge impacts on their expectation of UX.

[the remark this answers is not in the document]
Comment (joepavitt) [reply design/design-thinking.md:16]: The faces were added after the details/age.

[the remark this answers is not in the document]
Comment (joepavitt) [reply design/design-thinking.md:16]: In the grand scheme of personas, we have trimmed other pieces that I would expect to cover, e.g. marital status, location: https://www.interaction-design.org/literature/topics/personas

for me though, age is a very important factor that I don't want to exclude. The heart of creating user personas is about being able to build empathy with that user. In over generalising, empathy becomes more difficult to attain, as we should have enough detail in there to make the persona someone that we can relate to and feel is 'real".

[the remark this answers is not in the document]
Comment (joepavitt) [reply design/design-thinking.md:16]: If we were doing Design Thinking in its fullest, we would have done months of user research prior to any of these exercises or development, we obviously don't have scope to do that. What I am trying to do here is pull out the most valuable components of design thinking (given the speed at which we are having to deliver and restrictions in resource) to ensure we create a product/UX that empathises with users.

**Pull request merged**